EventKit/eventkit-cloud
# Updating dependencies and npm version

# Description of Improvement

- NPM version has been updated to 8.19.2
- The image of node has been updated 16.17.1
- Various dependency versions have been updated or locked to the same version
- Image numbering bumped to avoid clashes

## How to test

Run tests and verify that everything still works. You can check version numbers by running `npm --version` and `npm ls <package>` 

## Quality Assurance

The following items have been updated:
